src/context-response.json

```diff
@@ -0,0 +1,629 @@
+{
```

Comment:
This data produces the following chart:

![](https://user-images.githubusercontent.com/558581/106573984-ef59a600-64ff-11eb-8d58-d51981d2fbbd.png)